### DISCUSSION TITLE
Time crate upgrade in 4.x

### DISCUSSION THREAD

**Discussion by Antiarchitect** [actix/actix-web#2382] [Ideas] [answered]
Old `time 2.x` crate is using unmaintained `stdweb` and that is shown in `cargo-audit` output. Is there any chance to move to `time 3.x` in the upcoming `actix-web 4.x`?

**Comment by fakeshadow** [comment, marked as answer]
or use httpdate which is a 1.0 version dep that also used by hyper and remove homebrew http date parse

**Reply by robjtede** [reply]
Good idea. It was far easier to take this method than to try using the new set up in time 0.3. For one, it is not possible (or at least not at all obvious how) to parse two digit years, causing the RFC850 format to become unsupported.

Upgrade PR: #2383
